Solutions/Patching/Omnia-Classic/1.0/turnoffvms/README.MD

```diff
@@ -0,0 +1,24 @@
+# Update Management - Turn Off VMs
+
+## Intended usage
+
+This script is intended to be run as a part of Update Management Pre/Post scripts.
+
+## Requirements
+
+It requires:
+
+- A valid RunAsAccount.
```

Comment:
What permissions does it require? Could be a good idea to document.

Reply:
Permissions needed added to readme. Readme moved to suggested new location.